content/automate/ManageIQ/System/Policy.class/__methods__/vmcloudreconfigure_request_auto_approve.yaml

```diff
@@ -0,0 +1,13 @@
+---
+object_type: method
+version: 1.0
+object:
+  attributes:
+    name: vmcloudreconfigure_request_auto_approve
```

Comment:
@billfitzgerald0120 
Is there a corresponding PR for a method (internal or external) that would take advantage of this new attribute?

Reply:
This method yaml should have been included in https://github.com/ManageIQ/manageiq-content/pull/205.